### PR TITLE
DP-947 Fix Bugs in Opening a Chart in Superset

### PR DESCRIPTION
**Pull request by yuguang**
When clicking on a table name to go to the "Explore" view in Superset, there is a Javascript error which prevents the page from loading.

### CODE REVIEW

File: superset/assets/src/explore/components/ExploreChartPanel.jsx [hunk 1]
```diff
@@ -89,10 +90,12 @@ class ExploreChartPanel extends React.PureComponent {
 
   renderChartDescription() {
     let description;
-    if (this.props.slice) {
+    if (this.props.slice && this.props.slice.description) {
       description = this.props.slice.description;
-    } else {
+    } else if (this.props.table_name) {
```
Comment (qv-pat) [inline superset/assets/src/explore/components/ExploreChartPanel.jsx:95]: good catch

File: superset/connectors/sqla/views.py [hunk 2]
```diff
@@ -367,44 +367,6 @@ def create(self):
         db.session.commit()
         return jsonify(success=True)
 
-    @expose('/<pk>', methods=['GET', 'POST'])
```
Comment (qv-pat) [inline superset/connectors/sqla/views.py:370]: I'll try and do some debugging on my end to figure out why this view wasn't working when I have the time

**Review by qv-pat — APPROVED** [review]
Feels sort of odd to have the docker-compose changes in this PR, but otherwise looks good to me

**Comment by yuguang** [comment]
I did not put the docker compose changes in a branch earlier while I was working on the JIRA issue. Let's just merge in the changes to keep the repo up to date.

**Pull request merged**